gandalf.js

```diff
@@ -1,66 +1,67 @@
-const formattedDate = new Date().toLocaleDateString();
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.